sql/fxa_users_daily_v1.sql

```diff
@@ -18,14 +18,29 @@ CREATE TEMP FUNCTION udf_mode_last(x ANY TYPE) AS ((
   LIMIT 1
 ));
 
+CREATE TEMP FUNCTION udf_contains_tier1_country(x ANY TYPE) AS ((
```

Comment:
could you also put this at `udf/udf_contains_tier1_country`?

Reply:
I'm opposed to that because the particular country values differ based on context. FxA data has long-form country names whereas telemetry uses two-letter country codes. We adjust the FxA countries to match telemetry, but only at the last step before presenting results to users.

Reply:
sounds good, could you put that in a comment?

Reply:
So this function is not really applicable to other contexts.